src/pages/buy.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { PopupContainer } from '@components/popup/container';
+import { useChangeScreen } from '@common/hooks/use-change-screen';
+import { Header } from '@components/header';
+import { Caption, Text, Title } from '@components/typography';
+import { Button, Flex, Stack } from '@stacks/ui';
+import { Link } from '@components/link';
+import { ScreenPaths } from '@common/types';
+import AddFunds from '@assets/images/add-funds.svg';
+import { SpaceBetween } from '@components/space-between';
+import { useCurrentAccount } from '@store/accounts/account.hooks';
+import { transakUrl } from '@common/transak-helper';
+
+export const BuyPage = () => {
```

Comment:
Could we split this up into two components, this one, and a `buy.layout.tsx` ?

Reply:
Sure will do.